e2e-tests/gef/cypress/integration/scenarios/Deal editors cannot submit to UKEF/maker-should-not-submit-its-own-submitted-deals.spec.js

```diff
@@ -1,14 +1,15 @@
-import relative from '../relativeURL';
-import applicationDetails from '../pages/application-details';
-import automaticCover from '../pages/automatic-cover';
-import applicationSubmission from '../pages/application-submission';
-import CREDENTIALS from '../../fixtures/credentials.json';
-import applicationPreview from '../pages/application-preview';
-import returnToMaker from '../pages/return-to-maker';
-import statusBanner from '../pages/application-status-banner';
+import relative from '../../relativeURL';
```

Comment:
can we update the file name as well for this file to match the others from this folder?

Reply:
done

Reply:
done